kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -766,6 +734,43 @@ public class TypeSpec private constructor(
 
     @Suppress("RedundantOverride")
     override fun addTypes(typeSpecs: Iterable<TypeSpec>): Builder = super.addTypes(typeSpecs)
+
+    @Suppress("RedundantOverride")
```

Comment:
Do we need to keep these overrides for backwards compatibility? Should we mark them as "delete in 2.0"?

Reply:
That's right. We have so many that you can't even see they're already in a comment region which does indicate they're only for compatibility so we should know they can be deleted for 2.0.